godump.go

```diff
@@ -277,6 +263,21 @@ func (d *Dumper) Dd(vs ...any) {
 	exitFunc(1)
 }
 
+// clone creates a copy of the Dumper with the same configuration.
+func (d *Dumper) clone() *Dumper {
+	var newDumper = *d
+	return &newDumper
+}
+
+// format applies the configured formatter to the string with the given color code.
+func (d *Dumper) format(code, str string) string {
```

Comment:
"format" is ambiguous and could have many implied meanings - I'd very much prefer to keep this as "colorize"